example/src/screens/Room.tsx

```diff
@@ -1,181 +1,187 @@
+import { BrandColors } from '@colors';
+import { BackgroundAnimation } from '@components/BackgroundAnimation';
 import { Icon } from '@components/Icon';
+import { RoomParticipant } from '@components/RoomParticipant';
+import { Typo } from '@components/Typo';
 import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
 import { RootStack } from '@model/NavigationTypes';
 import type { NativeStackScreenProps } from '@react-navigation/native-stack';
-import React, { useState, useEffect, useCallback } from 'react';
-import { StyleSheet, Text, View, Pressable } from 'react-native';
+import React, { useCallback, useEffect } from 'react';
+import { StyleSheet, View, Pressable, Dimensions } from 'react-native';
+import { useVideoroomState } from 'src/VideoroomContext';
 
-import { Controls } from '../Controls';
-import { Settings } from '../Settings';
+import { CallControls } from '../components/CallControls';
+
+const HEADER_AND_FOOTER_SIZE = 126;
+const PADDING_BETWEEN_PARTICIPANTS = 16;
```

Comment:
shouldn't it be 8px? https://www.figma.com/file/lVPaILYT73AJIOsoXNbsXj/Membrane_Videoroom_Ready_Designs?node-id=628%3A18043&t=972uWMzRl47USrm4-0

Reply:
It is in the actual css, this constant is used for calculating size of the video tiles and using 8px per row wasn't enough to get tiles to fit, looking at this right now I agree that the naming is bad, I can change it though.

Reply:
ok, please change the naming then, now it's really misleading to me

Reply:
Renamed.